node/node.go

```diff
@@ -399,6 +399,7 @@ func (node *AlgorandFullNode) startMonitoringRoutines() {
 	go node.oldKeyDeletionThread(node.ctx.Done())
 
 	if node.config.EnableUsageLog {
+		node.monitoringRoutinesWaitGroup.Add(1)
```

Comment:
there is not corresponding `monitoringRoutinesWaitGroup.Done()`

Reply:
Ah you're right. It should be passed in via `go logging.UsageLogThread(node.ctx, node.log, 100*time.Millisecond, node.monitoringRoutinesWaitGroup)`. I'm curious why we ended up passing nil instead.